governance.qmd

```diff
@@ -0,0 +1,119 @@
+A moderator may vet contributions and decide to admit or remove contributed packages from the repositories in accordance with the Review Policy.
+
+### Administrator
+
+An administrator is a member of the R-Multiverse Management Committee (RMC).
+Administrators have write access to all code repositories under the `r-multiverse` GitHub organization and maintain the infrastructure for the project.
+They have the right to cast votes on all proposals on community-related decisions, the right to appoint moderators, and the right to change the composition of the RMC itself.
+The RMC as a whole is the entity that controls the project, and is responsible for the continued shaping of this governance document and other policies, including the Review Policy and Code of Conduct.
+
+## R-Multiverse Management and Collaboration
+
+The R-Multiverse project is managed through a collaborative, consensus-based approach.
+
+### Communication
+
+Communication is always done publicly via GitHub, and all decisions related to the project are made in public.
+An exception exists for confidential security and conduct-related matters, for which there is a private GitHub repository accessible by administrators only.
+The use of this private repository ensures that even confidential conversations remain accountable and auditable.
+Private mailing lists are not used by the project.
+
+### R-Multiverse Management Committee
+
+The R-Multiverse Management Committee (RMC) is responsible for the active management of the R-Multiverse project.
+The main role of the RMC is to further the long-term development and health of the community as a whole, and to ensure that balanced and broad-based user engagement and contributor participation takes place.
+
+As part of this, the RMC is the sole proposer and approver of changes to this governance document and all policies regarding operation of the `r-multiverse` GitHub repositories.
+
+The RMC collectively appoint and delegate authority to moderators to perform a limited subset of duties.
+
+### Proposals
+
+All changes to this governance document, or to a policy document such as the Code of Conduct or Review Policy that governs how the repositories operate will be posted for comment at the `r-multiverse/help` repository as a Proposal.
+All members of the community are invited to participate, and Proposals may be withdrawn or revised by their initial proposer as a result.
+Revisions likely to be regarded as material shall be proposed as a new Proposal, with reference to the preceding discussion.
+
+Proposals will be deemed to have been adopted if both one calendar month has passed after notice has been publicly posted and either (i) there has been no further response from an administrator or (ii) dissent by an administrator was recorded and the Proposal has been passed after voting has taken place as per the Voting Process described below.
```

Comment:
Maybe it would be clearer to say "(i) there has been no dissent from an administrator" instead of "(i) there has been no further response from an administrator". Otherwise, the trivial case of explicit unanimity seems to fall through the cracks as worded.

Reply:
Good catch, thanks! Amended in [7467dce](https://github.com/r-multiverse/r-multiverse.github.io/pull/16/commits/7467dce293fc1b359f5e3d1bb017d36d8da4ba8e).